### ISSUE TITLE
Crash when I choose certain options of dialogue with Tandi

### ISSUE THREAD

**Issue by ShoundDawa** [alexbatalov/fallout1-ce#76]
i apologize for my poor english handling, but i'm new to reporting bugs

I use Fallout - CE with Lutris and besides having infinite black screens after a success in first aid and some crashes when using F5.

Find that in some Tandi dialogue options, the game crashes no matter what:

¿What else is around theses parts?
¿Have you heard any stories from down south?

Log in Lutris showed me this:
 
Started initial process 683 from gamemoderun ./fallout-ce
Start monitoring process.
*** buffer overflow detected ***: terminated
Monitored process exited.
Initial process has exited (return code: 134)
All processes have quit
Exit with return code 134

I use this resolution:
Set the Fullscreen resolution here.
SCR_WIDTH=854
SCR_HEIGHT=480

System: Lubuntu 23.04
Build February 2023 (version 1.0) - fallout-ce-linux-x64.tar.gz 
Game: Fallout 1, GOG

if anyone needs more information, feel free to ask me anything

**Comment by alperkose** [comment]
Similar issue was reported in #43 and #20. It looks like it's resolved with this issue #26 

The fix is not in the released build but you can build yourself (since the non-released builds are deleted by Github after a while). There are a couple of ways I found, both hacky and can be improved but I didn't spend enough time:

### 1. build it yourself
Clone this repository, install dependencies and build it from the source code:

```
sudo apt install git cmake clang build-essential libsdl2-dev zlib1g-dev
git clone https://github.com/alexbatalov/fallout1-ce.git && cd fallout1-ce
 cmake -B build -D CMAKE_BUILD_TYPE=RelWithDebInfo && cmake --build build -j $(nproc)
```
If everything works fine, you can find the executable in the `build` folder

###  2. use docker
If you have docker installed, create a file named `Dockerfile` with the following content:
```
FROM ubuntu:22.04

RUN apt update
RUN apt install -y git cmake clang build-essential libsdl2-dev zlib1g-dev

WORKDIR /builder
RUN git clone https://github.com/alexbatalov/fallout1-ce.git && cd fallout1-ce
CMD cmake -B build -D CMAKE_BUILD_TYPE=RelWithDebInfo && cmake --build build -j $(nproc)
```
I would advise creating the file in an empty folder for better performance (docker build pushes the contents of the folder to builder) then run following commands:
```
docker build -t fallout1-ce-builder . 
docker run --name=fallout-builder fallout1-ce-builder 
docker cp fallout-builder:/builder/fallout1-ce/build/fallout-ce ./fallout-ce
```

### 3. Fork the repo and run github-actions in your own repository
You might have to make some changes (enable actions from forked repository) and maybe push a commit to your own repository to trigger actions

**Comment by alexbatalov** [comment]
Thanks for patience, download latest release.
